src/components/Footer/Footer.scss

```diff
@@ -12,3 +12,7 @@
   padding: 4em 0;
   color: white;
 }
+
+.love {
```

Comment:
can move this class into the `.footer-inner` class so the scss structure matches the markup structure